lib/modules/manager/hermit/artifacts.ts

```diff
@@ -0,0 +1,271 @@
+import pMap from 'p-map';
+import upath from 'upath';
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type { ExecOptions } from '../../../util/exec/types';
+import { localPathIsSymbolicLink, readLocalSymlink } from '../../../util/fs';
+import { getRepoStatus } from '../../../util/git';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+import type { ReadContentResult } from './types';
+
+/**
+ * updateArtifacts runs hermit install for each updated dependencies
+ */
+export async function updateArtifacts(
+  update: UpdateArtifact
+): Promise<UpdateArtifactsResult[] | null> {
+  const { packageFileName } = update;
+  logger.debug({ packageFileName }, `hermit.updateArtifacts()`);
+
+  try {
+    await updateHermitPackage(update);
+  } catch (err) {
+    const execErr: UpdateHermitError = err;
+    logger.warn(
+      { stdout: execErr.stdout, stderr: execErr.stderr },
+      `error updating hermit packages.`
+    );
```

Comment:
```suggestion
    logger.debug({ err }, `error updating hermit packages.`);
```

Artifact error will cause a warning / error on worker level